server/cowin/utils.py

```diff
@@ -0,0 +1,16 @@
+def age_filter(available_centers, age=None):
+    if age is None:
+        return available_centers
+
+    filtered = []
+    for center in available_centers:
+        f_sessions = []
+
+        for session in center["sessions"]:
+            if session["min_age_limit"] == int(age):
```

Comment:
```suggestion
            if session["min_age_limit"] <= int(age):
```
Just in case more slabs are introduced, say 30?

Reply:
This isn't needed, as pointed out by you :o